extensions/wikia/Wall/maintenance/CleanupCommentsIndex.php

```diff
@@ -0,0 +1,44 @@
+<?php
+require_once( __DIR__ . '/../../../../maintenance/Maintenance.php' );
+
+class CleanupCommentsIndex extends Maintenance {
+
+	public function __construct() {
+		parent::__construct();
+
+		$this->addOption( 'dry-run', 'only checks how many rows is affected' );
+	}
+
+	public function execute() {
+		global $wgDBname;
+
+		$db = wfGetDB( DB_MASTER );
+
+		$ids = $db->selectFieldValues(
+			[ 'page', 'comments_index' ],
+			'comment_id',
+			[ 'page_id=comment_id', 'page_namespace not in (1,500,501,1200,1201,2000,2001)' ],
```

Comment:
Should we remove `NS_TALK` from this whitelist? According to https://github.com/Wikia/app/blob/dev/extensions/wikia/ArticleComments/classes/CommentsIndex.class.php#L582 comments index entries should only be created for Wall or Forum content.